### ISSUE TITLE
Taboo: Change yaml word list format to have a dictionary of words + more

### ISSUE THREAD

**Issue by sbirmi** [lambdaFooBar/bari#44]
- Don't allow duplicates of the same word (with different disallowed words)
- Once this is done, also remove _usedWordIdxs. WordSet users shouldn't care about it at all. Instead, this should be tracked per WordSet requestor inside the WordSet. The requestor should perhaps only provide it's own game-id (or some other unique key).
